src/wallet/rpcdump.cpp

```diff
@@ -1761,8 +1762,8 @@ RPCHelpMan listdescriptors()
                     }},
                     {RPCResult::Type::NUM, "next", true, "The next index to generate addresses from; defined only for ranged descriptors"},
                 }},
-            }
-        },
+            }}
```

Comment:
suggested fix line 1758

s/Whether this is internal or external descriptor/Whether this is an internal or external descriptor/

Reply:
Fixed